docs/Licenses.txt

```diff
@@ -0,0 +1,251 @@
+ Apache License
```

Comment:
I think we should add headers to both the licenses in this file to be explicitly clear what they apply to, similar to example on https://softwareengineering.stackexchange.com/questions/234511/what-is-the-best-practice-for-arranging-third-party-library-licenses-paperwork